tests/rules/import-alias.test.ts

```diff
@@ -307,10 +357,9 @@ function runTests(platform: "win32" | "posix") {
                     },
                 ],
             },
-
-            // relative path overridden for a specified directory and exports from sibling module
+            // relative path overridde for a specified directory, exporting a sibling module
             {
-                code: `export { Potato } from "./rules/potato";`,
+                code: `export { Potato } from "./rules/foo";`,
```

Comment:
Unnecessary edit

Reply:
same code string are test key but i can revert that